storage-provider-guide/README.md

```diff
@@ -8,4 +8,4 @@ Within the EthStorage network, storage providers operate storage nodes (a.k.a. e
 
 The storage providers must provide proof of replication over time to collect rewards. This process is sometimes called "mining", and the storage provider is called a "miner".
 
-Becoming a storage provider in the EthStorage network is fully permissionless, provided the hardware requirements are satisfied.
+Participation as a storage provider is currently limited to whitelisted operators who meet the hardware requirements.
```

Comment:
During the initial launch, participation as a storage provider is limited to whitelisted operators. Once the network reaches stability, it will transition to a fully permissionless model.